File: OpenControl/Core/Network/DTO/Auth/Requests/RegisterUserRequest.swift
//
//  RegisterUserRequest.swift
//  OpenControl
//
//  Created by Vladislav Panev on 21.05.2023.
//

import Foundation

struct RegisterUserRequest: Codable {
    let firstName: String
    let lastName: String
    let email: String
    let mobilePhone: String
    let password: String
    let repeatPassword: String
    let ditSecurityQuestion: String?
    let ditSecurityAnswer: String?
    let middleName: String?
}

File: Back-end/Leader02.Api/Models/Requests/Auth/RegisterUserRequest.cs
namespace Leader02.Api.Models.Requests.Auth;

public class RegisterUserRequest
{
    public string FirstName { get; set; } = string.Empty;
    public string LastName { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public string MobilePhone { get; set; } = string.Empty;
    public string Password { get; set; } = string.Empty;
    public string RepeatPassword { get; set; } = string.Empty;
    public string? DitSecurityQuestion { get; set; }
    public string? DitSecurityAnswer { get; set; }
    public string? MiddleName { get; set; }
}